src/text-editor-presenter.coffee

```diff
@@ -1109,6 +1104,8 @@ class TextEditorPresenter
   setLineHeight: (lineHeight) ->
     unless @lineHeight is lineHeight
       @lineHeight = lineHeight
+      unless @scrollTop?
```

Comment:
:art: I think here we're checking if the position has been already restored: if it hasn't, we calculate it only the first time the line height is set.

Checking if `@scrollTop` has been defined, though, is somewhat misleading in this context: what would you think about extracting a method that tells this story a bit more explicitly? Something like: `::shouldRestoreScrollTop()`, `::shouldRestoreScrollLeft()`. Maybe we could also add another method that wraps `::updateScrollTop`, and call it `::restoreScrollTop()` (or `::deserializeScrollTop`) so that the code reads like:

```coffee
@restoreScrollTop() if @shouldRestoreScrollTop()
@restoreScrollLeft() if @shouldRestoreScrollLeft()
```